hadoop-plugin/src/main/resources/sample-dependency-pattern.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+
+<!--
+  Copyright 2015 LinkedIn Corp.
+
+  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+  use this file except in compliance with the License. You may obtain a copy of
+  the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+  License for the specific language governing permissions and limitations under
+  the License.
+-->
+
+
+<!-- This is a sample dependencyPattern file. All the dependencies of the project will be checked against the regular
+expression defined in <group>, <name> and <version>. Note that the dependencyPattern will be checked if all the <group>,
+<name> and <version> matches the patterns. The severity can take values ERROR, WARN and INFO and the message
+to go with the severity can be defined in the <message> tag. -->
+<dependencies>
```

Comment:
Why have it in XML . It is extremely verbose. Why not a form of gradle notation.